mjpeg-view/build.gradle

```diff
@@ -32,12 +32,8 @@ android {
 }
 
 dependencies {
-    testImplementation 'junit:junit:4.13.2'
     implementation 'androidx.appcompat:appcompat:1.4.0'
-    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
-    implementation 'androidx.annotation:annotation:1.3.0'
 
-    api 'io.reactivex:rxjava:1.3.8'
```

Comment:
how does it work without this? is it brought in as a dependency from `rxandroid`? if yes I think it should still be explicit then, since it's used directly

Reply:
It works !
Maybe it comes as a dependency from `api 'io.reactivex:rxandroid:1.2.1'`

Reply:
yup, I'm pretty sure that it's coming from `rxandroid`, but since it's used directly, as a best practice I would keep it explicit and revert this line.
I know that probably this is not the case, but it allows you to keep the `rxjava` version fixed even if you bump the other dependency, which might cause issues.

By the way, RxJava v1 is [EOL](https://github.com/ReactiveX/RxJava/tree/1.x) and we should stick with this strict version

Reply:
Btw, `rxJava` is on version 3 now https://github.com/ReactiveX/RxAndroid/releases/tag/3.0.0

Reply:
indeed, and It's very likely that bump it would probably break a lot of stuff

Reply:
the current version of rxandroid is obsolete too, but it's using an even [older](https://github.com/ReactiveX/RxAndroid/blob/1.x/rxandroid/build.gradle#L43) dependency of rxjava, so unless you have a strong opinion on this, I would prefer to restore it and use the latest dependency `'io.reactivex:rxjava:1.3.8'` of the v1.x before merge the pr